dashboard/src/t5gweb/templates/ui/stats.html

```diff
@@ -79,14 +79,14 @@ <h2>Overall Stats:</h2>
                         {
                         label: 'Incoming Cases',
                         data: {{ y_values['new_cases'] | safe }},
-                        borderColor: 'rgba(255, 206, 86, 1)',
-                        backgroundColor: 'rgba(255, 206, 86, 1)'
+                        borderColor: 'rgba(255, 0, 0, 1)',
```

Comment:
did you mean to make borderColor and backgroundColor different? that makes the `incoming cases` label greyish with a red border and the line is red w/ black dots